go/libraries/doltcore/env/dolt_docs.go

```diff
@@ -70,6 +74,24 @@ func (docs *Docs) Save(fs filesys.ReadWriteFS) error {
 			if err != nil {
 				return err
 			}
+		} else {
+			err := DeleteDoc(fs, doc.DocPk)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func DeleteDoc(fs filesys.ReadWriteFS, docName string) error {
+	for _, doc := range *AllValidDocDetails {
+		if doc.DocPk == docName {
+			path := getDocFile(doc.File)
+			exists, isDir := fs.Exists(path)
+			if exists && !isDir {
+				return os.Remove(path)
```

Comment:
You can't use os.Remove here, because that will only work with file systems backed by the os. You need fs.Remove

Reply:
Ohhhh gotcha. Changed it to fs.DeleteFile: https://github.com/liquidata-inc/dolt/pull/299/files#diff-9c4b80647736a52dd1d63681899240cbR91